tests/issues.targets

```diff
@@ -261,6 +261,9 @@
         <ExcludeList Include="$(XunitTestBinBase)\JIT\Methodical\tailcall_v4\smallFrame\smallFrame.cmd">
              <Issue>tail. call pop ret is only supported on amd64</Issue>
         </ExcludeList>
+        <ExcludeList Include="$(XunitTestBinBase)\JIT\opt\Tailcall\TailcallVerifyWithPrefix\TailcallVerifyWithPrefix.cmd" >
+            <Issue>x86 JIT doesn't support tail call opt</Issue>
```

Comment:
This test has tail.-prefixed calls, not implicit tail calls, so they are dispatched as tail calls even if FEATURE_FASTTAILCALL is not defined. I know for sure that the first failure in TailcallVerifyWithPrefix is because of `tail. call pop ret` sequence.